scipy/spatial/_spherical_voronoi.py

```diff
@@ -156,34 +159,38 @@ def __init__(self, points, radius=1, center=None, threshold=1e-06):
         if cKDTree(self.points).query_pairs(self.threshold * self.radius):
             raise ValueError("Duplicate generators present.")
 
-        radii = np.linalg.norm(self.points - self.center, axis=1)
-        max_discrepancy = np.abs(radii - self.radius).max()
+        radii: np.ndarray = np.linalg.norm(self.points - self.center, axis=1)
+        max_discrepancy: float = np.abs(radii - self.radius).max()
         if max_discrepancy >= self.threshold * self.radius:
             raise ValueError("Radius inconsistent with generators.")
-        self.vertices = None
-        self.regions = None
-        self._tri = None
+        self.vertices: np.ndarray
```

Comment:
The behavior has changed here-before these variables were initialized and now they are not. Was that intended?

Reply:
I did realize that, but it doesn't break any unit test. I think  `mypy` would complain about inconsistency between `None` and whatever happens later.

Reply:
The code was deleted when I merged Peter's recent PR for spherical Vor, so no change here now.